content/sensu-enterprise/3.3/integrations/email.md

```diff
@@ -290,7 +293,33 @@ type         | String
 required     | false
 example      | {{< highlight shell >}}"body": "/etc/sensu/email/body_template.erb"{{< /highlight >}}
 
+##### Example Templates
```

Comment:
We already have an example for the body here:

https://docs.sensu.io/sensu-enterprise/3.3/integrations/email/#custom-email-templates-example

Should this addition be removed and the subject addition added to where the above link is?